pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -64,7 +69,11 @@ def test_dotted_column_names(self, engine, connection):
         """When Hive returns a dotted column name, both the non-dotted version should be available
         as an attribute, and the dotted version should remain available as a key.
         """
-        row = connection.execute('SELECT * FROM one_row').fetchone()
+        row = connection.execute(text('SELECT * FROM one_row')).fetchone()
+
+        if sqlalchemy_version >= 1.4:
```

Comment:
let's stick to 1 pattern ==1.3 or >=1.4
I would prefer >=1.4

Reply:
done